### PR TITLE
Enhao CSSBattle

### PR DESCRIPTION
**Pull request by tinenhao**

### CODE REVIEW

File: src/CSSBattle/1_SimplySquare/tin.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div></div>
```
Comment (kiyonggg) [inline src/CSSBattle/1_SimplySquare/tin.html]: could not see battle #1 and #2, file named incorrectly

File: src/CSSBattle/3_PushButton/Tin.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<body>
+  <div class = rectangle>
```
Comment (kiyonggg) [inline src/CSSBattle/3_PushButton/Tin.html]: would be good to standardise the way you specify values for the class (i.e. class="rectangle" vs class = rectangle). putting quotation marks makes it more readable, more standardised and allows you to specify more than one class if necessary

same for battles 6 and 11

File: src/CSSBattle/8_ForkingCrazy/Tin.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<body>
+  <div class="container">
+    <div class="top">
+      <div class="stick" id="a"></div>
```
Comment (kiyonggg) [inline src/CSSBattle/8_ForkingCrazy/Tin.html]: id should be unique. if required more than once, consider using class instead

Comment (tinenhao) [reply src/CSSBattle/8_ForkingCrazy/Tin.html]: commit 362173e

File: src/CSSBattle/3_PushButton/Tin.html (hunk 2 above)
Comment (tinenhao) [reply src/CSSBattle/3_PushButton/Tin.html]: commmit 4ebbee1243e993add9f48d39646579cb86fc386b

File: src/CSSBattle/1_SimplySquare/tin.html (hunk 1 above)
Comment (tinenhao) [reply src/CSSBattle/1_SimplySquare/tin.html]: commit 6188481e167a19214605c9ad16a30a54c7a86507

**Review by kiyonggg — APPROVED** [review]
lgtm!

File: src/CSSBattle/3_PushButton/Tin.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    width: 150px;
+    position: fixed;
+  }
+
+  .outercircle {
+    margin-top: -50px;
+    margin-left: 25px;
+    background-color: #6592cf;
+    border-color: #6592cf;
+    border-width: 200px;
```
Comment (xinyee20) [inline src/CSSBattle/3_PushButton/Tin.html]: border-color and border-width can be removed, doesn't affect the circle

File: src/CSSBattle/4_UpsNDowns/Tin.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    border-radius: 0px 0px 50px 50px;
+  }
+
+  .purplebox {
+    width: 100%;
+    height: 50%;
+    background-color: #62306d;
+  }
+
+  #middlebase {
```
Comment (xinyee20) [inline src/CSSBattle/4_UpsNDowns/Tin.html:43]: Interesting use of invisible elements like purplebox and id to solve the challenge!

File: src/CSSBattle/5_AcidRain/Tin.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
+<body>
+  <div class="circle" id="c"></div>
+  <div class="circle" id="b"></div>
+  <div class="circle" id="a"></div>
+</body>
```
Comment (xinyee20) [inline src/CSSBattle/5_AcidRain/Tin.html:5]: I think a,b,c seems more intuitive than c,b,a. Can consider changing or even better, change the id names to make them easier to understand like the names you gave in Challenge 3 & 4

File: src/CSSBattle/6_MissingSlice/Tin.html [hunk 7]
```diff
@@ -0,0 +1,46 @@
+<body>
+  <div class="circle" id="c"></div>
+  <div class="circle" id="b"></div>
+  <div class="circle" id="a"></div>
+</body>
```
Comment (xinyee20) [inline src/CSSBattle/6_MissingSlice/Tin.html:5]: Likewise for the names here

**Review by xinyee20 — CHANGES_REQUESTED** [review]
Minor adjustments that can be made, else lgtm! [Review 2]

File: src/CSSBattle/3_PushButton/Tin.html (hunk 4 above)
Comment (tinenhao) [reply src/CSSBattle/3_PushButton/Tin.html]: commit bb4587c

File: src/CSSBattle/5_AcidRain/Tin.html (hunk 6 above)
Comment (tinenhao) [reply src/CSSBattle/5_AcidRain/Tin.html:5]: commit 1b7375d

File: src/CSSBattle/6_MissingSlice/Tin.html (hunk 7 above)
Comment (tinenhao) [reply src/CSSBattle/6_MissingSlice/Tin.html:5]: commit 1b7375d

**Review by xinyee20 — APPROVED** [review]
lgtm!

**Pull request merged**